src/Conversion/KrnlToLLVM/KrnlToLLVM.cpp

```diff
@@ -1486,6 +1501,81 @@ void mlir::populateAffineAndKrnlToLLVMConversion(RewritePatternSet &patterns,
   patterns.insert<KrnlUnaryMathOpLowering<KrnlTanOp>>(ctx);
 }
 
+void mlir::checkConstantOutputs(
+    ModuleOp &module, SmallVectorImpl<bool> &constantOutputs) {
+  Operation *entryPointOp;
+  auto walkResult = module->walk([&](mlir::Operation *op) -> WalkResult {
+    if (llvm::dyn_cast<KrnlEntryPointOp>(op)) {
+      entryPointOp = op;
+      return WalkResult::interrupt();
+    } else
```

Comment:
[nit]: The else is not necessary because the `then` part will return, suggest: 
```
if (llvm::dyn_cast<KrnlEntryPointOp>(op))i {
      entryPointOp = op;
      return WalkResult::interrupt();
}

return  WalkResult::advance();